tests/README.rst

```diff
@@ -79,7 +79,7 @@ Logs from the run are stored in the ``tests`` folder.
 
 For debugging of the container please run the container as::
 
-    make container-ci CONTAINER_TEST_ARGS="--rm -ti -v .:/anaconda:Z --entrypoint /bin/bash"
+    make container-ci CONTAINER_TEST_ARGS="-it --entrypoint /bin/bash"
```

Comment:
Please look here https://github.com/rhinstaller/anaconda/pull/2925

We have to decide the correct solution here.

Reply:
OK, let's do that in #2925. I dropped that commit from this PR.